azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/compute/options/AzureTemplateOptions.java

```diff
@@ -32,28 +32,11 @@
  */
 public class AzureTemplateOptions extends TemplateOptions implements Cloneable {
 
-   private String virtualNetworkName;
-   private String subnetId;
    private AvailabilitySet availabilitySet;
    private String availabilitySetName;
    private List<DataDisk> dataDisks = ImmutableList.of();
    private String resourceGroup;
-
-   /**
-    * Sets the virtual network name
-    */
-   public  AzureTemplateOptions virtualNetworkName(String virtualNetworkName) {
```

Comment:
do we need to deprecate them first or because it is still in labs we can change the public api this way?